### ISSUE TITLE
New reference for Section 5 Answer Engineering

### ISSUE THREAD

**Issue by jshin49** [pfliu-nlp/NLPedia-Pretrain#6]
First of all, I really appreciate all the work done to keep this thing up and running. I like the concept of a community-driven and maintained survey paper.

In our new work that has recently been accepted to ACL 2022, we propose a prompt-based few-shot method that should fall into the category of Answer Engineering.

More specifically, we map Dialogue State Tracking (DST) outputs into human-readable summaries. Although the process is automated through templatization, the summary templates themselves are hand-crafted.

Basically, we use a Summarization model to learn synthetic summaries on Task-oriented Dialogue datasets and then perform heuristic extraction of the slots.

Here's the link to the [code](https://github.com/jshin49/ds2) and the [paper](https://arxiv.org/abs/2203.01552).
